generator/generator-opc-generator/generators/app/templates/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack --mode production",
     "start": "webpack-dev-server",
     "test": "npm run build && jest --verbose=true",
-    "test:debug": "npm run build && SET DEBUG_MODE=1 && jest --verbose=true"
+    "test:debug": "npm run build && DEBUG_MODE=1 && jest --verbose=true"
```

Comment:
@gisumit why did u resolve this conversation??? this isn't resolved yet. I asked to remove && after DEBUG_MODE=1

Reply:
removes &&